packages/core-mobile/app/new/hooks/useSeedlessManageMFA.ts

```diff
@@ -0,0 +1,50 @@
+import { TotpChallenge } from '@cubist-labs/cubesigner-sdk'
+import { showSnackbar } from 'new/utils/toast'
+import useVerifyMFA from 'seedless/hooks/useVerifyMFA'
+import SeedlessService from 'seedless/services/SeedlessService'
+import Logger from 'utils/Logger'
+
+function useSeedlessManageMFA(): {
```

Comment:
create the same hook here with new toast component, existing useSeedlessManageMFA uses old snackbar component.